src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+function Person({
+  person,
+}) {
+  let message = '';
+
+  if (!person.isMarried) {
+    message = 'I am not married';
+  }
+
+  if (person.isMarried && person.sex === 'm') {
+    message = `${person.partnerName} is my wife`;
+  }
+
+  if (person.isMarried && person.sex === 'f') {
+    message = `${person.partnerName} is my husband`;
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age ? (
```

Comment:
It'll be better to use `&&` instead of `? :`

Reply:
I chenged code.